### PR TITLE
Create Te-Stack.json

### PR DESCRIPTION
**Pull request by Te-Stack**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Te-Stack.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Quincy Oghenetejiri Ukumakube",
+  "bio": "Software Developer and Technical Writer"
```
Comment (sital002) [inline data/Te-Stack.json:3]:
```suggestion
  "bio": "Software Developer and Technical Writer",
```
Please add a comma here

**Comment by Te-Stack** [comment]
it seems some check are not working because i used form before but that it is not reflecting on my tublian profile so i decided to use this instead

**Comment by sital002** [comment]
> it seems some check are not working because i used form before but that it is not reflecting on my tublian profile so i decided to use this instead

Yes you are correct once you create profile using forms there is no point in creating it again with json. here is your profile link https://www.biodrop.io/Te-Stack.
You can close this Pull Request

**Comment by Te-Stack** [comment]
Okay but it has not yet reflected in my tublian profile

**Comment by sital002** [comment]
> it seems some check are not working because i used form before but that it is not reflecting on my tublian profile so i decided to use this instead

You can edit your biodrop profile using forms here is the quick link for it https://www.biodrop.io/docs/how-to-guides/editing-forms

**Comment by sital002** [comment]
I will close this Pull Request

**Pull request closed without merging**